server/api/lineitems.js

```diff
@@ -1,8 +1,23 @@
 const router = require('express').Router();
+const Product = require('../db/Product');
+const Order = require('../db/Order');
 module.exports = router;
 
-router.get('/', (req, res) => {
+router.get('/', (req, res, next) => {
+  Order.findById(req.params.orderId)
+    .then(order => res.send(order))
+})
+
+router.post('/', (req, res, next) => {
```

Comment:
I would suggest moving this to orders.js as well, since it's adding to Order, and using the Order Model